generators/app/templates/src/services/AnalyticsService.js

```diff
@@ -1,13 +0,0 @@
-import { createMiddleware as createAnalyticsMiddleware } from 'redux-beacon';
```

Comment:
same

Reply:
I don't want this service, we weren't even using it. There's a card for GA though: https://trello.com/c/xba9v0SJ/19-google-analytics-seleccionable